.github/workflows/build.yml

```diff
@@ -11,6 +11,8 @@ on:
 jobs:
   build-menu:
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
```

Comment:
what happens without this, just curious

Reply:
turns out I needed to add `packages: write` permission, otherwise it failed
and it works without `contents: write`